### ISSUE TITLE
Buggy example

### ISSUE THREAD

**Issue by kodonnell** [andialbrecht/sqlparse#208]
This does not work for me with version 0.1.18:

```
>>> import sqlparse
>>> sql = """  create procedure db.sp()
...   begin
...   
...   create table t1 select a from t where abc;
...     
...   create table t2 select
...     avg(a) as a,
...   std(a) as a,
...     avg(a) as a,
...   std(a) as a,
...     avg(a) as a,
...   std(a) as a,
...     avg(a) as a,
...   std(a) as a,
...     avg(a) as a,
...   std(a) as a,
...     avg(a) as a,
...   std(a) as a
...   from t ; 
...   
...   create table t3
...   select
...     a,  
...     0
...   from t;
...   
...   end; 
... """
>>> sqlparse.parse(sql)
```

This fixes it:

```
>>> sqlparse.parse(sql.replace('end; \n', 'end'))
```

and so does messing round with the sql statements (e.g. deleting one of the columns in `t2` or `t3`). I've had a quick look, and while I don't know the root of the cause, it might pay to add an `if tokens: ...` into `group_tokens` in `sql.py`.

**Comment by vmuriart** [comment]
@kodonnell what was the error you were getting? I just ran your example on the current master branch and it seems to be working. I may have addressed it while fixing a few other bugs.

**Comment by kodonnell** [comment]
Seems to be fixed with 0.1.19.

Since you asked, here's the error I was getting (which you should be able to replicate by installing 0.1.18):

```python
  File ".../lib/python3.5/site-packages/sqlparse/sql.py", line 375, in group_tokens
    idx = self.token_index(tokens[0])
IndexError: list index out of range
```

**Comment by andialbrecht** [comment]
Thanks for the feedback, @kodonnell!
